examples/hono/src/routes/toddlePage.ts

```diff
@@ -79,44 +81,54 @@ export const toddlePage = async (c: Context<HonoEnv>) => {
     },
     packageName: undefined,
     globalFormulas: {
-      formulas: c.env.project.files.formulas,
-      packages: c.env.project.files.packages,
+      formulas: project.files.formulas,
+      packages: project.files.packages,
     },
   })
+  const head = renderHeadItems(
+    getHeadItems({
+      url,
+      page: component,
+      files: project.files,
+      project: project.project,
+      context: formulaContext,
+      theme,
+    }),
+  )
   const { html: body } = await renderPageBody({
-    component: toddleComponent as any, // TODO: Fix typing
+    component: component,
     formulaContext,
     env: formulaContext.env as ToddleServerEnv,
     req: c.req.raw,
-    files: c.env.project.files,
+    files: project.files,
     includedComponents,
     evaluateComponentApis: async (_) => ({
       // TODO: Show an example of how to evaluate APIs - potentially using an adapter
     }),
     projectId: 'my_project',
   })
+  const charset = getCharset({
+    pageInfo: component.route?.info,
+    formulaContext,
+  })
   return c.html(
     html`<!doctype html>
       <html lang="${language}">
         <head>
-          <meta charset="UTF-8" />
-          <title>
-            ${applyFormula(page.route.info?.title?.formula, formulaContext) ??
-            page.name}
-          </title>
-          <!-- The rest of the <head> elements are coming soon 🤞 -->
-          <link
-            rel="stylesheet"
-            fetchpriority="high"
-            href="/_static/reset.css"
-          />
+          ${raw(head)}
           <style>
             ${styles}
           </style>
         </head>
         <body>
-          ${raw(body)}
+          <!-- ${c.req.raw.url} -->
```

Comment:
Is this to make some debugging easier?

Reply:
It was actually something we rendered internally, but I don't think it adds a lot of value tbh. I'll remove it